dart_test.yaml

```diff
@@ -1,39 +1,29 @@
-tags:
-  # DDC-related tags declared here, since you can't target the DDC using platform selectors:
-  # https://github.com/dart-lang/test/issues/652
+# Specify chrome and VM as default platforms for running all tests,
+# then let the `@TestOn()` annotations determine which suites actually run
+platforms:
+  - chrome
+  - vm
 
-  # Tests that should only be run in the dev compiler
-  ddc:
-    skip: "skipping DDC-specific tests until UIP-2410 is implemented"
-  # Tests that should NOT be run in the dev compiler
-  no-ddc:
+# Default concurrency of 4 for unit tests, integration preset will override
+concurrency: 4
 
 presets:
   dartdevc:
-    exclude_tags: no-ddc && mirrors
-    platforms:
-      - chrome
-      - vm
-
-  dart2js:
-    exclude_tags: ddc && mirrors
-    platforms:
-      - chrome
-
-  travis:
-    reporter: expanded
-
-  integration:
+    exclude_tags: no-ddc
     paths:
-      - test/integration/
+      - test/vm_tests/
+      - test/over_react_component_declaration_test.dart
+      - test/over_react_component_test.dart
+      - test/over_react_dom_test.dart
+      - test/over_react_util_test.dart
 
-  unit:
-    concurrency: 4
-    # TODO: Risk of getting out of date when running `pub run dart_dev gen-test-runner`.
-    # Can't use dart_dev for test at the moment because the -P flag is not supported
+  dart2js:
+    exclude_tags: ddc
     paths:
-      - test/vm_tests/
```

Comment:
That's a bummer, I thought the test runner would ignore the vm tests since they're annotated with `@TestOn('vm')` but I guess that's not the case.